### PR TITLE
readme: add performance graph

### PR DESCRIPTION
**Pull request by sirupsen**
The data comes from https://github.com/datafold/data-diff/pull/67 which I'll clean up significantly, then submit it for review. I will also need to add some instructions on how to use the benchmarking to generate these graphs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ there are few/no changes, but is able to output each differing row! By pushing
 the compute into the databases, it's _much_ faster than querying for and
 comparing every row.
 
+![Performance for 100M rows](https://user-images.githubusercontent.com/97400/174860361-35158d2b-0cad-4089-be66-8bf467058387.png)
+
+**†:** Downloading all row implementation is not optimal. It's a single-process,
```
Comment (erezsh) [inline README.md:29]: This sentence needs a bit of work.

Comment (sirupsen) [reply README.md:29]: Can you tell me what you think needs work about it?

Comment (erezsh) [reply README.md]: Maybe just rephrase it:

The Python implementation for downloading rows isn't optimal in some of the drivers. It's a single process....

Comment (sirupsen) [reply README.md]: There's more to it than that, because of the GIL, etc. I did another pass and will merge

**Pull request merged**